flattentool/input.py

```diff
@@ -580,18 +581,18 @@ def get_sheet_headings(self, sheet_name):
             return []
 
         if self.vertical_orientation:
-            return [cell.value for cell in worksheet.columns[skip_rows][configuration_line:]]
+            return [cell.value for cell in worksheet[_get_column_letter(skip_rows+1)][configuration_line:]]
```

Comment:
Add spaces around + just for consistency

Reply:
Yep. We might want to introduce proper flake8 checking for flatten-tool at some point.